test/bench_internal/execute_benchmarks.cpp

```diff
@@ -0,0 +1,80 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2020 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "span.hpp"
+#include <benchmark/benchmark.h>
+#include <memory>
+#include <vector>
+
+namespace
+{
+[[gnu::noinline]] auto init_locals_1(fizzy::span<const uint64_t> args, uint32_t local_count)
+{
+    std::vector<uint64_t> locals;
+    locals.reserve(args.size() + local_count);
+    std::copy_n(args.begin(), args.size(), std::back_inserter(locals));
+    locals.resize(locals.size() + local_count);
+    return locals;
+}
+
+[[gnu::noinline]] auto init_locals_2(fizzy::span<const uint64_t> args, uint32_t local_count)
+{
+    std::vector<uint64_t> locals(args.size() + local_count);
```

Comment:
Is this the same as `locals(args.size() + local_count, 0);` e.g. the one used in `execute.cpp`?

Reply:
Yes. The `0` value is default.